#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: Disable Help>Contents... menu item.
# Bug-Ubuntu: https://bugs.launchpad.net/bugs/323988
# Author: Daniel Hahler <ubuntu@thequod.de>

@DPATCH@
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/src/VBoxHelpActions.cpp virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/src/VBoxHelpActions.cpp
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/src/VBoxHelpActions.cpp	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/src/VBoxHelpActions.cpp	2009-10-07 13:44:10.275472897 +0200
@@ -33,6 +33,7 @@
 
     contentsAction = new QAction (aParent);
     contentsAction->setIcon (VBoxGlobal::iconSet (":/help_16px.png"));
+    contentsAction->setDisabled(true);
 
     webAction = new QAction (aParent);
     webAction->setIcon (VBoxGlobal::iconSet (":/site_16px.png"));
